oaknational/Oak-Web-Application
# Curriculum: Add curriculum landing page and subject-phase selector

## Description

- Added curriculum landing with placeholder text
- Added subject-phase selector component
- Added subject phase selector to homepage

## How to test

1. Go to [branch preview app](https://deploy-preview-1716--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum)
2. You should see the curriculum landing page with the subject-phase selector
3. Select a subject (History), phase (Secondary), and, if applicable, an Exam Board (AQA)
4. Verify that you can click the "View" button and see an alert about navigating to the chosen curriculum page
5. De-select the items by clicking on the buttons from step 3
6. Verify that you receive an appropriate error message when you click the "View" button


## Checklist
